config/sync/core.entity_form_display.node.article.default.yml

```diff
@@ -125,6 +126,16 @@ content:
       rows: 3
       placeholder: ''
     third_party_settings: {  }
+  field_tags:
+    type: entity_reference_autocomplete_tags
+    weight: 27
+    region: content
+    settings:
+      match_operator: CONTAINS
+      match_limit: 10
+      size: 60
+      placeholder: ''
+    third_party_settings: {  }
```

Comment:
Just marking this here. 

To add the _next_ tag after the first one, a comma is necessary. Is there a simple way to make the input automatically add this comma when a tag is selected clicked.

Reply:
Good idea. Perhaps.